localenv/mock-account-servicing-entity/remix.config.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
   // assetsBuildDirectory: "public/build",
   // serverBuildPath: "build/index.js",
   // publicPath: "/build/",
-  serverDependenciesToBundle: ['axios']
+  serverModuleFormat: 'cjs',
+  browserNodeBuiltinsPolyfill: { modules: { querystring: true } }
```

Comment:
To fix an error from trying to use a node lib in the browser:

> Node builtin "querystring" (imported by "app/lib/utils.ts") must be polyfilled for the browser. You can enable this polyfill in your Remix config, e.g. `browserNodeBuiltinsPolyfill: { modules: { querystring: true } }` [plugin browser-node-builtins-polyfill-plugin]

Going to try and re-implement the util with a URLSearchParams

Reply:
even better, I deleted `app/lib/utils.ts` entirely. Apparently it wasn't doing anything that `URLSearchParams` couldn't. https://github.com/interledger/rafiki/pull/2349/commits/29c4a24a291cab900017c697b18a04182cf45f24